scripts/mastodon_setup/provision/script/script.rb

```diff
@@ -17,10 +17,12 @@
 require "terraform_helper"
 require "skytap_client"
 require "uri"
+require "httplog"
```

Comment:
Is this what's causing all the HTTP requests to be written to stdout and displayed as script output? That seems fine for testing, but it displays sensitive metadata to the end user so we should probably remove it later.

Reply:
Agreed, will remove